torchaudio/transforms.py

```diff
@@ -67,6 +67,12 @@ class Spectrogram(torch.nn.Module):
             This argument is only effective when ``power=None``. It is ignored for
             cases where ``power`` is a number as in those cases, the returned tensor is
             power spectrogram, which is a real-valued tensor.
+
+    Example
+
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalization=True)
+        >>> transformed_spectrogram = torchaudio.transforms.Spectrogram(n_fft = 800)(waveform)
```

Comment:
Hi @harishsdev. Could you assign the transform parameter and move the forward method to a separate line? You can see #1644 as an example. 
``` suggest 
transform = torchaudio.transforms.Spectrogram(n_fft = 512)
spectrogram = transform(waveform)
```